src/main/java/com/org/jenkins/Custom/Jenkins/Distribution/Service/generators/PackageConfigGenerator.java

```diff
@@ -0,0 +1,51 @@
+package com.org.jenkins.Custom.Jenkins.Distribution.Service.generators;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PackageConfigGenerator {
+
+
+    public static String generatePackageConfig(JSONObject formData) throws JSONException {
+
+        //Get the bundle object
+        JSONObject bundle = formData.getJSONObject("bundle");
```

Comment:
I can imagine that you'll want to pull this section out into a method or a class that focuses on simply generating the information behind the war.  It doesn't hurt to pull this into a method so it's easier to test.

Reply:
Done :+1: